www/custom_styles.css

```diff
@@ -58,4 +58,35 @@
     /* Softer green */
     border: 1px solid #000;
     /* Black border */
+}
+
+
+/*Styles for the Key Data boxes on home page*/
+.keyBoxes .info-box {
+    min-height: 0;
+    min-width: 80px;
```

Comment:
The box styling looks great! Could you make them a bit bigger?

Reply:
working on it

Reply:
do you want the text bigger or just the boxes ?

Reply:
@Konkaseeti Actually with the edits you already made I think it looks quite nice. I think the size is fine as is. Thanks!